R/runCalibration.R

```diff
@@ -735,19 +812,16 @@ runCalibrationOptim <- function(path,
 #' @importFrom dplyr %>% .data left_join mutate select
 #'
 .initStepSize <- function(i, stepSizeParams, outerObjective, stepSizeInit) {
-  if (i != 1) {
-    stepSizeParams <- stepSizeParams %>%
-      left_join(outerObjective, by = c("reg", "loc", "typ", "inc", "ttot")) %>%
-      mutate(stepSize = ifelse(
-        .data[["delta"]] > 0.001,
-        pmax(stepSizeInit, (.data[["fPrev"]] - .data[["f"]]) / .data[["delta"]]),
-        stepSizeInit
-      )) %>%
-      select(-"f", -"fPrev")
-  } else {
-    stepSizeParams <- mutate(stepSizeParams, stepSize = stepSizeInit)
-  }
-  return(stepSizeParams)
+  stepSizeParams %>%
+    left_join(outerObjective, by = c("reg", "loc", "typ", "inc", "ttot")) %>%
+    mutate(stepSize = ifelse(
+      i != 1 & .data[["delta"]] > 0.001,
+      pmax(stepSizeInit, (.data[["fPrev"]] - .data[["f"]]) / .data[["delta"]]),
+      stepSizeInit
```

Comment:
Probably entirely subjective but I find this easier to read:
```suggestion
      i == 1 | .data$delta <= 0.001,
      stepSizeInit,
      pmax(stepSizeInit, (.data$fPrev - .data$f) / .data$delta)
```